chart/values.yaml

```diff
@@ -705,6 +705,12 @@ obs:
         # NodePort associated with https port
         https: 90010
 
+storageClass:
+  enabled: true
+  nameSuffix: single-replica
+  default: false
+  repl: 1
```

Comment:
Wonder if it'd be useful to have this under a parameters key?
CC @niladrih  @Abhinandan-Purkait

Reply:
Hi ! It could help with keeping the values tidy.
I can fix this if you want, just let me know.

Reply:
I just chatted to @Abhinandan-Purkait, yes please place parameters inside parameters key. Thank you!

Reply:
Done ✅